lib/hawkular/inventory/inventory_api.rb

```diff
@@ -429,6 +429,17 @@ def create_metric_type(feed_id, metric_type_id, type = 'GAUGE', unit = 'NONE', c
       MetricType.new(new_mt)
     end
 
+    # List operation types for given resource type
+    # @param [String] resource_type_path canonical path of the resource type entity
+    # @return [Array<String>] List of operation type ids
+    def list_operation_types(resource_type_path)
+      parsed_path = CanonicalPath.parse(resource_type_path)
+      feed_id = parsed_path.feed_id
+      resource_type_id = parsed_path.resource_type_id
+      ret = http_get("/feeds/#{feed_id}/resourceTypes/#{resource_type_id}/operationTypes")
+      ret.map { |ot| ot['id'] }
```

Comment:
Because there is a conflict in the nomenclature, the `name` is reserved for the string that should be displayed to user, while id is some internal thing. Or at least that's how it was designed. Often, the 2 are the same though, it depends what agent reports to the inventory as `id` and `name`.

check https://git.io/vrqE5 to see how the data look like